meta-mender-tegra/meta-mender-tegra-common/recipes-bsp/tegra-binaries/tegra-storage-layout_%.bbappend

```diff
@@ -1,7 +1,8 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-SRC_URI:append:p3768-0000-p3767-0000 = " \
```

Comment:
Hi @RoopanJK,

This is the only line which I'm a bit unsure about. As the `flash_l4t_t234_nvme_rootfs_ab.xml` is now unconditionally added, does it possibly overwrite a file of the same name for other boards too? Otherwise should be good.